r/tests/testthat/test-python.R

```diff
@@ -17,6 +17,8 @@
 
 context("To/from Python")
 
+skip("for now")
```

Comment:
Need to remove this before we merge, of course

Reply:
✔️